python/python/Ice/Util.py

```diff
@@ -17,36 +16,43 @@
 from .Properties import Properties
 
 if TYPE_CHECKING:
-    import Ice
+    import asyncio
+
+    from Ice import Communicator, Identity, InitializationData, Properties, ToStringMode
 
 __name__ = "Ice"
 
 
 @overload
-def initialize() -> Ice.Communicator: ...
+def initialize() -> Communicator: ...
 
 
 @overload
-def initialize(args: list[str]) -> Ice.Communicator: ...
+def initialize(args: list[str]) -> Communicator: ...
+
 
+@overload
+def initialize(args: list[str], initData: InitializationData) -> Communicator: ...
 
 @overload
-def initialize(args: list[str], initData: Ice.InitializationData) -> Ice.Communicator: ...
+def initialize(args: None = None,  initData: InitializationData = InitializationData()) -> Communicator: ...
```

Comment:
`= InitializationData()` Is this safe?

Reply:
I  think so, because `@overloads` are only use for type-hints. 

Another alternative would be `initData: InitializationData | None = None`

Reply:
If they're only used for type hints how does it work?